packages/cloud-cognitive/src/components/NonLinearReading/NonLinearReading.js

```diff
@@ -86,7 +84,7 @@ NonLinearReading.displayName = componentName;
 // See https://www.npmjs.com/package/prop-types#usage.
 NonLinearReading.propTypes = {
   /**
-   * The term of the component appears as a pill.
+   * The keyword of the component appears as a pill.
```

Comment:
Design settled on a way to refer to the "term": "keyword". All JS and CSS have been updated to reflect this.